playground/docs.config.ts

```diff
@@ -7,4 +7,30 @@ export default defineDocsConfig({
   redirects: {
     '/foo': '/bar',
   },
+  // theme: {
+  //   50: '#FEFDF7',
+  //   100: '#FDFCEF',
+  //   200: '#FAF6D6',
+  //   300: '#F7F1BD',
+  //   400: '#F2E78C',
+  //   500: '#ECDC5A',
+  //   600: '#D4C651',
+  //   700: '#8E8436',
+  //   800: '#6A6329',
+  //   900: '#47421B',
+  //   950: '#2F2C12',
+  // }
+  theme: {
```

Comment:
It would be much much cooler if we could generate this pallet. Even if less perfect, I prefer less configuration for end docs projects. (doc system os a tool rather than a website builder so consider end user (like me) dummy enough that only knows how to pass one/two color codes optionally)

Reply:
Also let's consider this is currently an opinionated theme for UnJS projects. we probably need less variation between docs.

We might try https://github.com/unjs/theme-colors

Reply:
yes! Will add unjs/theme-colors and simplify process (colors are already generated using this tool!)